### ISSUE TITLE
package gtetrinet for distros?

### ISSUE THREAD

**Issue by oupala** [tatankat/gtetrinet#4]
Hi, if your version a gtetrinet is working on latest Gnome Shell distros, what would you think of publishing packages for these distro in order to make gtetrinet more popular?

**Comment by tatankat** [comment]
That would be nice, be my guest :)

**Comment by oupala** [comment]
I do confirm that your version of Gtetrinet is building fine with checkinstall (compiled on debian unstable).

I think it would be great to publish it into debian repository.

**Comment by ebourg** [comment]
I've built gtetrinet on Debian and intend to upload the package soon. It works really great, many thanks all those involved in the GTK 3 port!

**Comment by oupala** [comment]
@ebourg Are you a debian maintainer?

I'm asking that to you because the original author of gtetrinet is also a debian maintainer and he is the official packager for the [gtetrinet package](https://packages.debian.org/stretch/gtetrinet).

**Comment by ebourg** [comment]
@oupala Yes I'm a Debian developer (and also the author of Jetrix). The gtetrinet package was removed from Debian 10 because it didn't work with GTK 3. I've uploaded it again with the GTK 3 changes and hopefully it'll be part of Debian 11. I'll also backport it to Debian 10 later.

**Comment by oupala** [comment]
It is so great to see that gtetrinet will be back in debian and Ubuntu...

However, it is currently [not yet available in debian repository](https://packages.debian.org/search?suite=all&searchon=names&keywords=gtetrinet) (even in *sid* and *experimental*).

I am so impatient to be able to install `gtetrinet` in debian stable from `buster-backports`!

**Comment by ebourg** [comment]
The package is queued and awaiting review before landing in sid:

https://ftp-master.debian.org/new/gtetrinet_0.7.11+git20200916.46e7ade-1.html

I'll let you know when it's available.

**Comment by tatankat** [comment]
@ebourg You missed the party.
Tetrinet is now again available in Debian buster-backports, bullseye & sid [:tada:] (https://packages.debian.org/search?suite=all&searchon=names&keywords=gtetrinet) 
Which distro is next?
@ebourg Thanks for your work!

**Comment by tatankat** [comment]
About those other distro's: is it ok for you (@ebourg) if I change the directory where the binary is installed to the default binary dir (which is $(prefix)/bin) using bin_PROGRAMS in the Makefile?
That way, anyone who wants (at least Debian, but not openSUSE nor Gentoo), can change that directory at configure time using simply `--bindir`

**Comment by ebourg** [comment]
The updated package has also propagated to Ubuntu (Hirsute Hippo). I think it should reach Linux Mint in the near future too.

No objection to the install dir change, I'll adapt the package as necessary.

**Comment by tatankat** [comment]
Nice!
I've pushed the change (the changes are some patches from other distro's I do now upstream).

**Comment by b10n** [comment]
I just created an [AUR package](https://aur.archlinux.org/packages/gtetrinet-gtk3-git/) for Arch.

**Comment by ebourg** [comment]
@b10n what about replacing the [gtetrinet](https://aur.archlinux.org/packages/gtetrinet/) package instead? I guess it no longer works now.

**Comment by b10n** [comment]
That one can still be built, although it pulls in a quite a few dependencies that are not (anymore) in the official repositories.
